crates/cli/src/utils/mod.rs

```diff
@@ -150,6 +152,13 @@ pub fn parse_delay(delay: &str) -> Result<Duration> {
     Ok(delay)
 }
 
+// Parses a `AccessList` from a &str
+pub fn parse_access_list(s: &str) -> Result<AccessList> {
```

Comment:
hmm, I am wondering if we should add some additional formats for better UX here? eg fallback to comma-separated addresses list or something similar

Reply:
I like that idea, I'll try my best to implement it.

Reply:
let me know if any questions!